Description: Don't install manpage with setup.py.
 Use debhelper to install manpage so we can use update-alternatives to
 manage the manpages.
Author: Charlie Smotherman <cjsmo@cableone.net>

--- a/setup.py
+++ b/setup.py
@@ -102,7 +102,7 @@
     license = 'GNU LGPL',
     keywords = 'test unittest doctest automatic discovery',
     url = 'http://readthedocs.org/docs/nose/',
-    data_files = [('man/man1', ['nosetests.1'])],
+    #data_files = [('man/man1', ['nosetests.1'])],
     package_data = {'': ['*.txt',
                          'examples/*.py',
                          'examples/*/*.py']},
